### DISCUSSION TITLE
What is "atomic CSS"

### DISCUSSION THREAD

**Discussion by Kazuhiro-Mimaki** [chakra-ui/panda#989] [Q&A]
I cannot understand "atomic CSS" in the [docs](https://panda-css.com/docs/overview/getting-started).

> Panda is a styling engine that generates styling primitives to author atomic CSS and recipes in a type-safe and readable manner.

What is "atomic CSS"?

**Comment by anubra266** [comment]
https://medium.com/@DaphneWatson/atomic-css-15ade4083fb7#:~:text=%E2%80%9CAtomic%20CSS%20is%20the%20approach,the%20idea%20of%20reusable%20components.
